packages/ui/src/TextArea.module.scss

```diff
@@ -0,0 +1,89 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.container {
+  @include c.typographyBodySmall;
+
+  color: c.$colorText;
+  box-sizing: border-box;
+  margin: 0;
+
+  .label {
+    margin-bottom: c.$grid;
+  }
+
+  .helperText {
+    display: block;
+    margin: c.$grid * 2 0 0 c.$grid * 2;
+  }
+
+  .errorContainer {
+    margin-top: c.$grid;
+  }
+
+  .textAreaContainer {
+    display: flex;
+    flex-direction: row;
+  }
+
+  textarea {
+    display: block;
+    padding: c.$grid * 2 c.$grid * 3;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    box-sizing: border-box;
+    width: 100%;
+    font-family: inherit;
+    font-weight: inherit;
+    font-size: inherit;
+    line-height: inherit;
+    letter-spacing: inherit;
+    color: inherit;
+    border-radius: c.$borderRadius;
+    background-color: c.$colorNeutralLighter;
+
+    &::placeholder {
+      color: c.$colorTextSecondary;
+    }
+
+    // Note: Resize is not supported by IE11, it's presence does no harm though
+    &.notResizable {
+      /* doiuse-disable css-resize */
+      resize: none;
+      /* doiuse-enable css-resize */
+    }
+  }
+
+  &.empty {
+    color: c.$colorTextSecondary;
+  }
+
+  &:hover,
+  &.hover {
+    textarea {
+      border-color: c.$colorPrimary;
+    }
+  }
+
+  &.disabled {
+    textarea {
+      color: c.$colorTextSecondary;
+      background-color: c.$colorNeutral;
+      border-color: c.$colorNeutralLight;
+    }
+  }
+
+  &.hasError {
+    textarea {
+      border-color: c.$colorError;
+    }
+  }
+
+  textarea:focus {
```

Comment:
I am wondering if we have to make the outline a bit more complicated because of https://hazelcast.slack.com/archives/C01BT11F24E/p1605077912095500
I would suggest to try:
1. Add an absolute positioned element to the .textAreaContainer
2. Use `:focus-within` on that element to draw the outline there
3. Leave the `:focus` styling of the outline for IE11 as it does not support `:focus-within`
4. To make it disappear in other browser add `:focus` override for the parent that have `:focus-within`

WDYT?